src/Controller/ConsumptionController.php

```diff
@@ -23,41 +24,44 @@ class ConsumptionController extends AbstractController
      * @param PaginatorInterface $paginator
      * @return array
      */
-    public function index(Request $request, PaginatorInterface $paginator): array
+    public function index(Request $request, PaginatorInterface $paginator)
     {
-        $consumptions = $this->getDoctrine()->getRepository(Consumption::class)->findBy([
-            'user' => $this->getUser()
-        ], [
-            'dateTime' => 'DESC'
-        ]);
+        $templateData = $this->getTemplateData($request, $paginator);
 
-        $consumptions = $paginator->paginate(
-            $consumptions,
-            $request->query->getInt('page', 1),
-            5
-        );
+        if($request->get('pdf')) {
```

Comment:
Is dit error proof (bijvoorbeeld als pdf niet in request staat?

Reply:
je drukt op de knop. die stuurt een request met "pdf" erin